package.json

```diff
@@ -3,27 +3,27 @@
   "version": "0.0.0-alpha.25",
   "description": "A JavaScript library for interacting with @daostack/arc ethereum smart contracts",
   "main": "dist/arc.js",
-  "types": "lib/arc.d.ts",
+  "types": "index.d.ts",
   "files": [
     "dist/",
-    "lib/",
     "docs/",
     "test/",
     "migrations/",
     "migrated_contracts/",
     "config/",
+    "index.d.ts",
     "package-scripts.js",
     "package-scripts/",
     "truffle.js",
     ".babelrc",
     ".eslintignore",
-    ".eslintignore",
     ".eslintrc"
   ],
   "scripts": {
     "start": "nps",
     "test": "npm start test",
-    "build": "npm start build"
+    "build": "npm start build",
+    "lint": "npm start lint"
```

Comment:
does this run also tslint ? 
take a look at https://github.com/daostack/arc.js/blob/tslint/package.json#L27

Reply:
Yes, it runs both ts and eslint.  You can how that is implemented [here](https://github.com/daostack/arc.js/blob/5d9737392e4c1bec8c928503d274ca9ea9cee62e/package-scripts.js#L48)